flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/TableTestProgram.java

```diff
@@ -177,6 +177,14 @@ public List<FunctionTestStep> getSetupFunctionTestSteps() {
                 .collect(Collectors.toList());
     }
 
+    /** Convenience method to avoid casting. It assumes that the order of steps is not important. */
```

Comment:
I usually faced adj + noun rather than noun + noun... not sure how correct it is...
```suggestion
    /** Convenient method to avoid casting. It assumes that the order of steps is not important. */
```